data/json/monsters/zed_misc.json

```
@@ -1,4 +1,85 @@
 [
+  {
+    "id": "mon_frog_mother",
+    "type": "MONSTER",
+    "name": { "str": "frog mother" },
+    "description": "A large, bulbous creature with an extremely large mouth and small, stubby legs with no feet.  It is covered in an oily greenish-black residue and large, white pustules that seem ready to pop.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "240 L",
+    "weight": "300 kg",
+    "hp": 600,
+    "speed": 75,
```

```suggestion
    "speed": 60,
```
With the proposed ranged attacks and things, this would be a cool zombie to have moving more slowly but getting into nasty business at range.